src/schoology-extractor/schoology_extractor/helpers/arg_parser.py

```diff
@@ -55,16 +55,38 @@ def parse_main_arguments(args_in: list) -> MainArguments:
 
     assert isinstance(args_in, list), "Argument `args_in` must be a list"
     parser = ArgParser()
-    parser.add('-k', '--client-key', required=True, help='Schoology client key.', type=str)  # type: ignore
-    parser.add('-s', '--client-secret', required=True, help='Schoology client secret.', type=str)  # type: ignore
-    parser.add('-g', '--grading-period', required=True, help='CSV with the grading periods.', type=str)  # type: ignore
+    parser.add(  # type: ignore
+        '-k',
+        '--client-key',
+        required=True,
+        help='Schoology client key.',
+        type=str,
+        env_var="CLIENT_KEY")
+
+    parser.add(  # type: ignore
+        '-s',
+        '--client-secret',
+        required=True,
+        help='Schoology client secret.',
+        type=str,
+        env_var="CLIENT_SECRET")
+
+    parser.add(  # type: ignore
+        '-g',
+        '--grading-period',
+        required=True,
+        help='CSV with the grading periods.',
+        type=str,
+        env_var="GRADING_PERIOD")
```

Comment:
Shouldn't this be `SCHOOLOGY_GRADING_PERIODS`?
```suggestion
        env_var="SCHOOLOGY_GRADING_PERIODS")
```

Reply:
✅